File: website/src/code/entities/Website.ts
export class Website {
    constructor(data: any) {
        this.id = data['id'];
        this.name = data['name'];
        this.founders = data['founders'];
        this.yearStarted = data['yearStarted'];
        this.yearEnded = 'yearEnded' in data? data['yearEnded'] : null;
        this.url = data['url'];
        this.country = data['country'];
        this.isActive = data['isActive'];
        this.type = data['type'];
    }

    public id: number;
    public name: string;
    public founders: Array<string>;
    public yearStarted: number;
    public yearEnded?: number;
    public url: string;
    public country: string;
    public isActive: boolean;
    public type: string;
}

File: server/DataAPIs/Entities/Website.cs
namespace VideoGameArchive.Entities
{
    public class Website
    {
        public Website() {}

        public int id { get; set; }
        public required string name { get; set; }
        public string? founders { get; set; }
        public int yearStarted { get; set; }
        public int? yearEnded { get; set; }
        public required string url { get; set; }
        public required string country { get; set; }
        public bool isActive { get; set; }
        public required string type { get; set; }
        public int? lastWebcrawlDate { get; set; }
    }
}